platform.go

```diff
@@ -97,18 +97,6 @@ func (p *platformPlugin) handleClipboardGetData(arguments interface{}) (reply in
 }
 
 func (p *platformPlugin) handleWindowSetTitle(arguments interface{}) (reply interface{}, err error) {
-	appSwitcherDescription := struct {
-		Label        string `json:"label"`
-		PrimaryColor int64  `json:"primaryColor"`
-	}{}
-	err = json.Unmarshal(arguments.(json.RawMessage), &appSwitcherDescription)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to decode arguments")
-	}
-	p.glfwTasker.Do(func() {
-		p.window.SetTitle(appSwitcherDescription.Label)
```

Comment:
I think we should still do this.

Reply:
By default, `appSwitcherDescription.Label` always returns the `name` field of pubspec.yaml.

Reply:
This field is for mobile devices, not suited for desktop: https://github.com/flutter/flutter/blob/f7a6a7906be96d2288f5d63a5a54c515a6e987fe/packages/flutter/lib/src/services/system_channels.dart#L79-L88

Reply:
The best we could have is to use: https://github.com/flutter/flutter/blob/7168421e9f8f4122bfe1036e2f398c6392dd9d17/packages/flutter/lib/src/widgets/route_notification_messages.dart#L30-L38
But this is currently only possible on web.
I'll open a PR.

Reply:
Alright! I never got that this was not the correct handling of the switcher description. Let's indeed handle it correctly and wait for the window-specific title setter method.